### DISCUSSION TITLE
Users management : Only from T0 ?

### DISCUSSION THREAD

**Discussion by captainmac40** [LoicVeirman/HardenAD#147] [Q&A] [answered]
Hello, it should be an dumb question but, according Tier model, DC should be on Tier 0 wright ? 
So to add an user to any OU you should use an Tier 0 admin account ?

It's a little bit confused for me, and my deployement is small (2 DC in 2 VM, approx 40 workstation and 60-80 users and only 2 admins Yes I use an bazooka to kill an bee by using tier model but why not :) )

Thanks !

**Comment by captainmac40** [comment, marked as answer]
As I say : Dumb question ... Answer is linked to delegations roles :)

**Comment by LoicVeirman** [comment]
You make my day \o/

**Reply by captainmac40** [reply]
You'r welcome :) 

C'est pas simple de faire la différence entre la séparation des Tiers (et donc des réseaux quelque part si j'ai bien compris les bonnes pratiques d’implémentation) et le moyen de se connecter :)

On utilise le même outil (les RSAT ou powershell) mais en fonction du compte utilisé on peut plus ou moins agir sur l'AD.

C'est quand même un "gros morceau" à assimiler en auto-formation l'active directory (heureusement qu'il y a IT Connect par exemple pour aider) mais c'est passionnant :)
